docs/src/content/en/docs/workflows/using-with-agents-and-tools.mdx

```diff
@@ -37,6 +37,19 @@ export const testAgent = new Agent({
 
 In this example, `step1` uses the `testAgent` to generate an interesting fact about the country based on a given city.
 
+Mastra agents use a default schema that expects a prompt string as input and returns a text string as output.
+
+```json
+{
+  inputSchema: {
+    prompt: string
+  },
+  outputSchema: {
+    text: string
+  }
+}
+```
```

Comment:
I just made some big changes to Agents and Tools, i think this part should be added though. I can either create a PR or, are you able to resolve the merge conflicts @justindeguzman?